### PR TITLE
Fix a typo and grammar and add a clarification.

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: core/src/nodes/handled_node_tasks.rs [hunk 1]
```diff
@@ -58,7 +58,7 @@ use void::Void;
 
 /// Implementation of `Stream` that handles a collection of nodes.
 pub struct HandledNodesTasks<TInEvent, TOutEvent, THandler> {
-    /// For each active task, a sender allowing to transmit messages. Closing the sender interrupts
+    /// An FNV hashmap for each active task to a sender allowing to transmit messages. Closing the sender interrupts
```
Comment (dvdplm) [inline core/src/nodes/handled_node_tasks.rs]: Is the implementation – FNV hashmap – really relevant here? 
```suggestion
    /// A map between active tasks to an unbounded sender, used to control the task. Closing the sender interrupts
```

File: core/src/nodes/handled_node_tasks.rs [hunk 2]
```diff
@@ -202,7 +202,7 @@ impl<TInEvent, TOutEvent, THandler> HandledNodesTasks<TInEvent, TOutEvent, THand
         self.tasks.keys().cloned()
     }
 
-    /// Provides an API similar to `Stream`, except that it cannot error.
+    /// Provides an API similar to `Stream`, except that it cannot produce an error.
```
Comment (dvdplm) [inline core/src/nodes/handled_node_tasks.rs]: I think this was fine as it was?

Comment (jamesray1) [reply core/src/nodes/handled_node_tasks.rs]: Error isn't a verb. You could say cannot fail, cannot err, cannot produce an error.

Comment (dvdplm) [reply core/src/nodes/handled_node_tasks.rs]: You are correct, my bad.

**Pull request merged**